cucumber-expressions/ruby/spec/cucumber/cucumber_expressions/custom_parameter_type_spec.rb

```diff
@@ -120,6 +136,28 @@ def ==(other)
           expect(transformed_argument_value).to eq(Color.new('dark red'))
         end
 
+        context "when a custom parameter with nested optional captures is specified" do
```

Comment:
I'd imagine this extra level of organization will harm test portability so this can be flattened out